aiida/schedulers/plugins/lsf.py

```diff
@@ -698,7 +701,7 @@ def _parse_time_string(self, string, fmt='%b %d %H:%M'):
                 thetime = datetime.datetime.strptime(actual_string, f'{actual_fmt} L')
         except Exception as exc:
             self.logger.debug(f'Unable to parse time string {string}, the message was {exc}')
-            raise ValueError('Problem parsing the time string.')
+            raise ValueError(f'Problem parsing the time string: `{string}`') from exc
```

Comment:
I took the liberty of adding a bit of information for this error message (although the information is being logged, I guess there is no harm in also including it here). I enclosed in inverted commas so that it would be more clear what exactly is the string (also added this for the other error on lines [678/679](https://github.com/aiidateam/aiida-core/pull/4832/files#diff-f3bdb5ef823a005d31025d9d0b851a50587737040d0208810bc3d6eaf2b806b5R678-R679)).

Let me know if you don't like it @sphuber, I can take it out.